File: JavaScript/Ex.10.js
let numeros = [
  [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
  [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
]

for(let l = 0; l < 3; l++){
  for(let c = 0; c < 2; c++){
    for(let lc = 0; lc < 3; lc++){
      numeros[l][c][lc] = prompt("Insira um número: ")
    }
  }
}
  

File: Python/Ex.10.py
numeros = [
  [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
  [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
]

for l in range(0, 3):
  for c in range(0, 2):
    for lc in range(0, 3):
      numeros[l][c][lc] = int(input("Insira um número: "))